### PR TITLE
Add debug tool for masks.

### PR DESCRIPTION
**Pull request by enricoliscio**
Now I will patiently check them all.

### CODE REVIEW

File: keras_maskrcnn/bin/debug.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+import cv2
+import numpy as np
+
+# Allow relative imports when being executed as script.
+if __name__ == "__main__" and __package__ is None:
+    sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', '..'))
+    import keras_retinanet.bin
+    __package__ = "keras_maskrcnn.bin"
+
+# Change these to absolute imports if you copy this script outside the keras_retinanet package.
```
Comment (hgaiser) [inline keras_maskrcnn/bin/debug.py]: Change the comment from `keras_retinanet` to `keras_maskrcnn`.

File: keras_maskrcnn/bin/debug.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+from keras_retinanet.utils.transform import random_transform_generator
+from keras_retinanet.utils.visualization import draw_annotations, draw_boxes
+from keras_retinanet.utils.colors import label_color
```
Comment (hgaiser) [inline keras_maskrcnn/bin/debug.py]: Move the `keras_retinanet` imports above the comment.

File: keras_maskrcnn/bin/debug.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+            transform_generator=transform_generator
+        )
+    else:
+        raise ValueError('Invalid data type received: {}'.format(args.dataset_type))
+
+    return generator
+
+
+def parse_args(args):
+    parser     = argparse.ArgumentParser(description='Debug script for a Mask-RCNN network.')
```
Comment (hgaiser) [inline keras_maskrcnn/bin/debug.py]: Maybe this should be `'Debug script for a retinanet-maskrcnn dataset.'` instead (yeah yeah I know, it's like this in retinanet too ;))

File: keras_maskrcnn/bin/debug.py [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+    subparsers = parser.add_subparsers(help='Arguments for specific dataset types.', dest='dataset_type')
+    subparsers.required = True
+
+    coco_parser = subparsers.add_parser('coco')
+    coco_parser.add_argument('coco_path',  help='Path to dataset directory (ie. /tmp/COCO).')
+    coco_parser.add_argument('--coco-set', help='Name of the set to show (defaults to val2017).', default='val2017')
+
+    def csv_list(string):
```
Comment (hgaiser) [inline keras_maskrcnn/bin/debug.py]: The CSV and OID parsers are useless.

File: keras_maskrcnn/bin/debug.py (hunk 1 above)
Comment (enricoliscio) [reply keras_maskrcnn/bin/debug.py]: done

**Pull request merged**